openid-4-verifiable-presentations-1_0.md

```diff
@@ -432,54 +432,30 @@ Note: "https://self-issued.me/v2" is a symbolic string and can be used as an `au
 
 ## Verifier Metadata Management {#client_metadata_management}
 
-The `client_id_scheme` enables deployments of this specification to use different mechanisms to obtain and validate metadata of the Verifier beyond the scope of [@!RFC6749]. The term `client_id_scheme` is used since the Verifier is acting as an OAuth 2.0 Client.
+If the Client is not pre-registered with the Wallet, the `client_id` MUST be a URI.
```

Comment:
We should be prepared for the eventuality that people may say that this is an unnecessary change, since for the x509_san_dns case, the client_id currently is a DNS name and not a URI.

What URI value are we suggesting that people use after this PR is merged when they were previously using a DNS name with x509_san_dns?

Reply:
That makes sense.